### PR TITLE
fix/register status 

### PR DESCRIPTION
**Pull request by alexstratos**

### CODE REVIEW

File: x/pot/handler.go [hunk 1]
```diff
@@ -122,6 +122,13 @@ func handleMsgSlashingResourceNode(ctx sdk.Context, k keeper.Keeper, msg types.M
 		}
 	}
 
-	err := k.SlashingResourceNode(ctx, msg.NetworkAddress, msg.WalletAddress, msg.Slashing, msg.Suspend)
-	return &sdk.Result{}, err
+	amt, err := k.SlashingResourceNode(ctx, msg.NetworkAddress, msg.WalletAddress, msg.Slashing, msg.Suspend)
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeSlashing,
+			sdk.NewAttribute(types.AttributeKeyWalletAddress, msg.WalletAddress.String()),
```
Comment (mdumaisqsnetwork) [inline x/pot/handler.go:129]: Could you add attributes for the networkAddress (p2p address) and the suspend boolean? These are the only two attributes we care about when receiving the event in relayd

File: x/pot/types/events.go [hunk 2]
```diff
@@ -5,6 +5,7 @@ const (
 	EventTypeVolumeReport      = "volume_report"
 	EventTypeWithdraw          = "withdraw"
 	EventTypeFoundationDeposit = "foundation_deposit"
+	EventTypeSlashing          = "slashing"
```
Comment (mdumaisqsnetwork) [inline x/pot/types/events.go:8]: maybe slashing_resource_node is better, so we can have a different event for meta node slashing

File: x/register/keeper/resource_node.go [hunk 3]
```diff
@@ -154,7 +154,8 @@ func (k Keeper) AddResourceNodeStake(ctx sdk.Context, resourceNode types.Resourc
 	}
 
 	resourceNode = resourceNode.AddToken(tokenToAdd.Amount)
-	resourceNode.Suspend = false
+	//resourceNode.Suspend = false
```
Comment (mdumaisqsnetwork) [inline x/register/keeper/resource_node.go:157]: If adding stake doesn't unsuspend the node, then how do we want resource nodes to unsuspend themselves? Should we add a new ppd command to unsuspend that would make SP node send a new slashing tx but with suspend=false and amount=0 ?

**Review by mdumaisqsnetwork — CHANGES_REQUESTED** [review]

File: x/pot/handler.go [hunk 4]
```diff
@@ -122,6 +122,15 @@ func handleMsgSlashingResourceNode(ctx sdk.Context, k keeper.Keeper, msg types.M
...
-	err := k.SlashingResourceNode(ctx, msg.NetworkAddress, msg.WalletAddress, msg.Slashing, msg.Suspend)
-	return &sdk.Result{}, err
+	amt, nodeType, err := k.SlashingResourceNode(ctx, msg.NetworkAddress, msg.WalletAddress, msg.Slashing, msg.Suspend)
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeSlashing,
+			sdk.NewAttribute(types.AttributeKeyWalletAddress, msg.WalletAddress.String()),
+			sdk.NewAttribute(types.AttributeKeyNodeP2PAddress, msg.NetworkAddress.String()),
+			sdk.NewAttribute(types.AttributeKeyAmount, amt.String()),
+			sdk.NewAttribute(types.AttributeKeySlashingNodeType, nodeType.String()),
```
Comment (mdumaisqsnetwork) [inline x/pot/handler.go:132]: Still needs an attribute for the suspend boolean. I wouldn't want to mark a PP node as suspended in SP if it's not suspended in stchain

Comment (alexstratos) [reply x/pot/handler.go:133]: updated

**Review by mdumaisqsnetwork — APPROVED** [review]

**Pull request merged**